app/src/main/java/com/github/libretube/helpers/ImportHelper.kt

```diff
@@ -45,15 +50,34 @@ object ImportHelper {
      * Get a list of channel IDs from a file [Uri]
      */
     @OptIn(ExperimentalSerializationApi::class)
-    private fun getChannelsFromUri(activity: Activity, uri: Uri): List<String> {
+    private fun getChannelsFromUri(activity: Activity, uri: Uri, supportedClient: SupportedClient): List<String> {
         return when (val fileType = activity.contentResolver.getType(uri)) {
             "application/json", "application/*", "application/octet-stream" -> {
-                // NewPipe subscriptions format
-                val subscriptions = activity.contentResolver.openInputStream(uri)?.use {
-                    JsonHelper.json.decodeFromStream<NewPipeSubscriptions>(it)
-                }
-                subscriptions?.subscriptions.orEmpty().map {
-                    it.url.replace("https://www.youtube.com/channel/", "")
+                when (supportedClient) {
```

Comment:
Currently, when the user imports a CSV file from YouTube they still get prompted to choose a format (e.g. FreeTube, NewPipe) although that won't actually be used.
I think it makes sense here to remove that file type conditions above and only allow the user to select the format they want to import. Hence, please also add a `YouTube` entry to the enum class containing the different available formats.